test/sqlite/getters.jl

```diff
@@ -402,6 +402,15 @@ end
 	@test GetDrugConceptIDs(drug_exposure_ids, sqlite_conn) == GetDrugConceptIDs(GetDrugExposures(test_ids, sqlite_conn), sqlite_conn)
 end
 
+@testset "GetVisitProcedure Tests" begin
+    
+    # GetVisitProcedure([4010253.0], sqlite_conn)
+     visited_person_id = [30.0, 72.0, 79.0, 90.0, 90.0, 102.0, 152.0, 182.0, 202.0, 245.0]
+     
+     reh = sort(VisitFilterPersonIDs([4010253.0], sqlite_conn)).person_id[1:10]
```

Comment:
Make sure to be using GetVisitProcedure here instead of VisitFilterPersonIDs for testing as that is the function we want to test here. :slightly_smiling_face: